crates/biome_js_analyze/tests/specs/nursery/noImportCycles/ignoreTypes.ts

```diff
@@ -0,0 +1,2 @@
+/* should not generate diagnostics */
+import type { Foo } from "./types.ts";
```

Comment:
Any chance to add more tests?

```ts
import { type Bar } from 'b.ts'
import * as Lorem from 'l.ts'

const lorem: Lorem = {};
```

Reply:
@ematipico Actually these aren't considered as type-only imports. The TypeScript compiler doesn't remove them if the `verbatimModuleSyntax` is enabled.